pkg/apis/intevents/v1alpha1/brokercell_validation.go

```diff
@@ -89,6 +89,13 @@ func (componentParams *ComponentParameters) ValidateAutoscalingSpecification(fie
 			}
 		}
 	}
+	// At least one of the autoscaling metrics should be specified
+	isAvgMemoryUsageSpecified := componentParams.AvgMemoryUsage != nil && *componentParams.AvgMemoryUsage != ""
+	if componentParams.AvgCPUUtilization == nil && !isAvgMemoryUsageSpecified {
+		invalidValueError := apis.ErrInvalidValue(nil, componentPath)
+		invalidValueError.Details = "At least one of the autoscaling metrics (AvgCPUUtilization, AvgMemoryUsage) should be specified"
+		fieldErrors = fieldErrors.Also(invalidValueError)
+	}
```

Comment:
Questionable. Maybe add a TODO with a link to an issue. We need to figure out whether we should allow disabling HPA.

Reply:
Makes sense, there are few ways to approach this. I have logged [the task](https://github.com/google/knative-gcp/issues/1632) and outlined the options I see.